db/migrate/20250318024953_copy_solved_topic_custom_field_to_discourse_solved_solved_topics.rb

```diff
@@ -3,12 +3,15 @@
 class CopySolvedTopicCustomFieldToDiscourseSolvedSolvedTopics < ActiveRecord::Migration[7.2]
   disable_ddl_transaction!
 
-  BATCH_SIZE = 5000
+  BATCH_SIZE = 10_000
 
   def up
+    max_id = DB.query_single("SELECT MAX(id) FROM topic_custom_fields").first
```

Comment:
```suggestion
    max_id = DB.query_single("SELECT MAX(id) FROM topic_custom_fields WHERE tc.name = 'accepted_answer_post_id'").first
```

Shouldn't we only filter for the max_id for `topic_custom_fields` with name `accepted_answer_post_id`?